index.js

```diff
@@ -1,17 +1,164 @@
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  },
+
+];
+
+const frameInfo = `
+  <p class="client">
+    canopy
+  </p>
+  <img src="snapshot-portfolio/Counter.png" alt="counter">
+  <p class="role">
+    back end dev
+  </p>
+  <img src="snapshot-portfolio/Counter.png" alt="counter">
+  <p class="year">
+    2015
+  </p>
+`;
+
+const cards = Object.keys(cardArray);
+
+cards.forEach((elem) => {
+  const card = document.createElement('div');
+  card.classList.add('card-work');
+
+  const imgContainer = document.createElement('div');
+  imgContainer.classList.add('snapshot-portfolio');
+  const img = document.createElement('img');
+  img.setAttribute('src', cardArray[elem].img);
+  imgContainer.appendChild(img);
+
+  const leftBlock = document.createElement('div');
+  leftBlock.classList.add('left-block');
+  const title = document.createElement('h2');
+  title.classList.add('work-title');
+  title.innerHTML = cardArray[elem].title;
+  const frame = document.createElement('div');
+  frame.classList.add('frame');
+  frame.innerHTML = frameInfo;
+  const para = document.createElement('p');
+  para.classList.add('para');
+  para.innerHTML = cardArray[elem].text;
+  const tags = document.createElement('ul');
+  tags.classList.add('tags');
+
+  cardArray[elem].tech.forEach((item) => {
+    const tagChild = document.createElement('li');
+    tagChild.innerHTML = item;
+    tags.appendChild(tagChild);
+  });
+  const buttonEl = document.createElement('button');
+  buttonEl.classList.add('btn');
+  buttonEl.innerHTML = 'see project';
+
+  const popup = document.createElement('div');
+  popup.classList.add('popup');
+
+  const header = document.createElement('h2');
+  header.classList.add('work-title');
+  header.innerHTML = cardArray[elem].title;
+
+  const popFrame = document.createElement('div');
+  popFrame.classList.add('frame');
+  popFrame.innerHTML = frameInfo;
+
+  const popContainer = document.createElement('div');
+  popContainer.classList.add('pop-container');
+  const popimg = document.createElement('img');
+  popimg.setAttribute('src', cardArray[elem].img);
+  popContainer.appendChild(popimg);
+
+  const lang = document.createElement('ul');
+  lang.classList.add('tags');
+
+  cardArray[elem].tech.forEach((item) => {
+    const tagChild = document.createElement('li');
+    tagChild.innerHTML = item;
+    lang.appendChild(tagChild);
+  });
+
+  const txt = document.createElement('p');
+  txt.classList.add('pop-para');
+  txt.innerHTML = cardArray[elem].description;
+
+  const btnLive = document.createElement('button');
+  btnLive.classList.add('btn');
+  btnLive.innerHTML = 'see live';
```

Comment:
This is great 💯 but just like you did for the other ones, this should be getting data from your `cardArray`, and instead of just a `button`, it can be either an `a` tag or a `button` wrapped in an `a` tag, that will allow you to use `href` and add a link to the live version of your portfolio. It can be a dummy link but the link must be there and working. 😉